src/main/kotlin/mods/eln/server/console/ElnConsoleHandler.kt

```diff
@@ -7,6 +7,7 @@ import net.minecraft.event.ClickEvent
 import net.minecraft.server.MinecraftServer
 import net.minecraft.util.ChatComponentText
 import java.lang.Exception
+import java.util.*
```

Comment:
Fully qualify

Reply:
... I didn't even tell IntelliJ to do any auto corrections in those files, fixed the warnings manually, how on Earth???  Might want to change the formatter options for the project?